md/release-notes.md

```diff
@@ -1,19 +1,96 @@
+
+<a id="readable-names"/>
+
+### [Readable UI Designer artifact names](ui-designer-overview.md#readable-page-name) in Bonita repositories
+New pages, forms, layouts, and fragments created with Bonita UI Designer now use the name given by the developer as IDs in the repository.
+Easier to find when they need to be updated.
+For existing artifacts, nothing has changed: their IDs is a UUID. You can switch them to the new mechanism by renaming them.
+
+### UI Designer artifacts last update date
+Last update date of all UI Designer artifacts is not updated with the use of the preview any more. Only when changes are made.
+
+### BDM meta-information for Portal Administrator
+Since installing or updating BDM requires BPM Services to be paused, such rights are only granted to the Technical User.
+However, since the Portal Administrator can now install and update the BDM Access Control file from the Administrator Portal, he can now get information on BDM status and last update date.
+
+### New layout for the Administrator Portal "User details" page
+All details of a user in the organization are now diplayed on an AngularJS page of the Administrator Portal, bringing still more consistency to the Administrator Portal.
+
+### Business objects relationships
+The default relationship type between business objects is now "agreggation". It was "composition" before, which led to some validation errors: a single object cannot be linked to several parents with a composition relationship, but it can with an aggregation relationship.
+
+### API error messages
+Improvements of messages in case of error on API calls have been done: explain the cause and give options for easier error recoveries.
+
+### Studio status bar
+At the bottom right hand corner of Bonita Studio, the Eclipse status bas has been added to give information about the back-end activities, especially in case of operations on remote repositories. In order not to lose diagram designing space, icons of the coolbar are now small by default. If you prefer them large, go to the studio Preferences menu.
+
+### Application descriptor "deploy"
+Just like for profiles or BDM access control, there is now a "Deploy" option in the Application descriptor menu of Bonita Studio.
+
+### Official signatures on Windows
+Bonita binaries as well as Studio installer are now officially signed on Windows OS, to prevent useless security alerts.
+
 <a id="techonolgy-updates"/>
 
+## Technology updates
+
+* The supported Tomcat version for this new release is Tomcat 8.5.31, in the embedded Bonita Portal as well as in the Tomcat bundle.
+* The supported Windows Server OS is now Windows Server 2016.
```

Comment:
⚠️ this was already the supported version in 7.6, so this should be removed
See https://documentation.bonitasoft.com/bonita/7.6/release-notes#toc15

Reply:
removed